app/admin/jobs/page.tsx

```diff
@@ -53,69 +53,76 @@ export default async function JobsPage() {
-            </tr>
-          </thead>
-          <tbody>
-            {routineJobsWithStringDates.map((job) => (
-              <tr key={job.id} className="hover:bg-gray-50">
-                <td className="px-6 py-4 border-b">{job.name}</td>
-                <td className="px-6 py-4 border-b">{job.cronExpression}</td>
-                <td className="px-6 py-4 border-b">
-                  {job.enabled ? (
-                    <span className="text-green-600 font-semibold">✓ Enabled</span>
-                  ) : (
-                    <span className="text-red-600 font-semibold">✗ Disabled</span>
-                  )}
-                </td>
-                <td className="px-6 py-4 border-b">
-                  <div className="flex gap-2">
-                    <Link
-                      href={`/admin/jobs/routine/${job.id}`}
-                      className="text-blue-600 hover:text-blue-800"
-                    >
-                      View Details
-                    </Link>
-                    {job.sandboxUrl && (
-                      <a
-                        href={job.sandboxUrl}
-                        target="_blank"
-                        rel="noopener noreferrer"
-                        className="text-green-600 hover:text-green-800"
-                      >
-                        Sandbox
-                      </a>
-                    )}
-                  </div>
-                </td>
+        <div className="overflow-x-auto">
+          <table className="min-w-full border border-gray-300">
+            <thead>
+              <tr className="bg-gray-500">
+                <th className="px-6 py-3 border-b text-left">Name</th>
+                <th className="px-6 py-3 border-b text-left">Cron Expression</th>
+                <th className="px-6 py-3 border-b text-left">Status</th>
+                <th className="px-6 py-3 border-b text-left">Actions</th>
               </tr>
-            ))}
-          </tbody>
-        </table>
+            </thead>
+            <tbody>
+              {routineJobsWithStringDates.map((job) => (
+                <tr key={job.id}>
+                  <td className="px-6 py-4 border-b">{job.name}</td>
+                  <td className="px-6 py-4 border-b">{job.cronExpression}</td>
+                  <td className="px-6 py-4 border-b">
+                    {job.enabled ? (
+                      <span className="text-green-600 font-semibold">✓ Enabled</span>
+                    ) : (
+                      <span className="text-red-600 font-semibold">✗ Disabled</span>
+                    )}
+                  </td>
+                  <td className="px-6 py-4 border-b">
+                    <div className="flex gap-2">
+                      <Link
+                        href={`/admin/jobs/routine/${job.id}`}
+                        className="text-blue-600 hover:text-blue-800 inline-block min-h-[44px] flex items-center"
+                        onClick={(e) => e.stopPropagation()}
```

Comment:
I'm seeing this error:
![image](https://github.com/user-attachments/assets/84a0ffc4-dc7a-4641-aaed-d5c2aa8d6650)

Idk where it's coming from, but I think it's from some of these onClick methods that we should abstract out into client component files